### PR TITLE
chore: removed 'picture' text from image alt

### LINKED ISSUE #365: (JS-0750) Prefer not to use words image, photo in image alt content
## Description
Enforce img alt attribute does not contain the word image, picture, or photo. Screenreaders already announce img elements as an image. There is no need to use words such as image, photo, and/or picture.

## Occurrences
There are 12 occurrences of this issue in the repository.

See all occurrences on DeepSource &rarr; [deepsource.io/gh/lenstube-xyz/lenstube/issue/JS-0750/occurrences/](https://deepsource.io/gh/lenstube-xyz/lenstube/issue/JS-0750/occurrences/)

### PR DESCRIPTION
**Pull request by zt4ff**
## Description

Closes #365 

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented on my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings

### CODE REVIEW

**Comment by zt4ff** [comment]
@sasicodes, I updated it based on the comment to use the channel handle.

So I used the `.handle` based on there the image `src` is coming from

**Comment by sasicodes** [comment]
@zt4ff you can just do alt={channel.handle} right? why template literal?

**Comment by zt4ff** [comment]
@sasicodes 
For the most part, the `.handle` have the `any` type. 

Typescript did not complain about i though

**Comment by sasicodes** [comment]
> @sasicodes For the most part, the `.handle` have the `any` type.
> 
> Typescript did not complain about i though

`handle` is a mandatory field in the profile. it should work I think 🤔 

do you see `ts` warnings?

**Comment by zt4ff** [comment]
> > @sasicodes For the most part, the `.handle` have the `any` type.
> > Typescript did not complain about i though
> 
> `handle` is a mandatory field in the profile. it should work I think thinking
> 
> do you see `ts` warnings?

No, I did not see any warning
Can I make the changes real quick?

**Comment by sasicodes** [comment]
> > > @sasicodes For the most part, the `.handle` have the `any` type.
> > > Typescript did not complain about i though
> > 
> > 
> > `handle` is a mandatory field in the profile. it should work I think thinking
> > do you see `ts` warnings?
> 
> No, I did not see any warning Can I make the changes real quick?

yes, please!

**Comment by zt4ff** [comment]
> > > > @sasicodes For the most part, the `.handle` have the `any` type.
> > > > Typescript did not complain about i though
> > > 
> > > 
> > > `handle` is a mandatory field in the profile. it should work I think thinking
> > > do you see `ts` warnings?
> > 
> > 
> > No, I did not see any warning Can I make the changes real quick?
> 
> yes, please!

@sasicodes I made a push

File: apps/web/src/components/Settings/BasicInfo.tsx [hunk 1]
```diff
@@ -278,7 +278,7 @@ const BasicInfo = ({ channel }: Props) => {
           }
           className="object-cover object-center w-full h-48 bg-white rounded-xl md:h-56 dark:bg-gray-900"
           draggable={false}
-          alt="Cover Image"
+          alt={channel.handle + ' Cover'}
```
Comment (sasicodes) [inline apps/web/src/components/Settings/BasicInfo.tsx:281]: can we use the template literal for this one?

**Review by sasicodes — CHANGES_REQUESTED** [review]

**Review by sasicodes — APPROVED** [review]
LGTM

File: apps/web/src/components/Settings/BasicInfo.tsx (hunk 1 above)
Comment (zt4ff) [reply apps/web/src/components/Settings/BasicInfo.tsx]: @sasicodes  done

**Pull request merged**